README.md

```diff
@@ -1,2 +1,54 @@
-Chameleon System
-================
+<p align="center">
+    <a href="https://chameleonsystem.com" target="_blank">
+        <img src="https://demo.chameleon-system.de/chameleon/mediapool/8/ae/chameleon-logo_id3834.png">
+    </a>
+    <br />
+    <span>
+        The E-Business Platform
+    </span>
+</p>
+
+Chameleon System is e-commerce and content management redefined: Flexible, based on Symfony, customer- and
+project-oriented. Chameleon is a product of the [ESONO AG](https://www.esono.de/).
+
+Demo
+----
+
+Find a demo at [https://demo.chameleon-system.de](https://demo.chameleon-system.de)
+
+Documentation
+-------------
+
+The user manual and technical documentation in English language are currently in progress.
+
+The user manual in German language is available here: [https://manual.chameleon-system.de](https://manual.chameleon-system.de)
+
+Installation
+------------
+
+TBA
```

Comment:
Work in progress - also this PR?

Reply:
Nope, this needs yet some work by @proligde and me (I propose PRs only if they are ready to merge from my view, or add "WIP" to the title).